react/features/analytics/handlers/AmplitudeHandler.js

```diff
@@ -0,0 +1,70 @@
+import amplitude from 'amplitude-js';
+
+import { getJitsiMeetGlobalNS } from '../../base/util';
+
+import AbstractHandler from './AbstractHandler';
+
+/**
+ * Analytics handler for Amplitude.
+ */
+class AmplitudeHandler extends AbstractHandler {
+    /**
+     * Creates new instance of the Amplitude analytics handler.
+     *
+     * @param {Object} options -
+     * @param {string} options.amplitudeAPPKey - The Amplitude app key required
+     * by the Amplitude API.
+     */
+    constructor(options) {
+        super();
+
+        const { amplitudeAPPKey } = options;
+
+        if (!amplitudeAPPKey) {
```

Comment:
```js
if(!options.amplitudeKey) ...
```
Why we need this kind of destruct here ?

Reply:
Looks prettier to me. Is there a downside?

Reply:
Not a big one, but you are creating a new variable for no reason.